### ISSUE TITLE
Various Loggers

### ISSUE THREAD

**Issue by narendasan** [acm-uiuc/arbor#18]
e.g Log to file

**Comment by narendasan** [comment]
Changes to /logger/log.go

**Comment by pattyjogal** [comment]
Hi! I would like to contribute to this issue, but I just had a question before I start:

Where is the best place to decide which logger to use/which file to log to? Would the user specify it when instantiating the arbor instance, or should it be in some settings class, or maybe on each call to logger (i.e. one function per logging method)?

 I only ask because the latter option sounds cumbersome for devs, but I don't know where an appropriate place to store this info would be.

Thanks!

**Comment by narendasan** [comment]
I would think probably the `config.go` file that users usually create when they make an arbor app @bcongdon thoughts?

**Comment by bcongdon** [comment]
Yeah, `config.go` sounds like the right place. This should be an optional setting

**Comment by pattyjogal** [comment]
Ok, thank you!
